Makefile

```diff
@@ -1,6 +1,6 @@
 export GIT_DIFF ?= off
 
-.SILENT:
+#.SILENT:
```

Comment:
Don't forget to get this back before merging :)

Reply:
Good catch